granary/bluesky.py

```diff
@@ -22,6 +23,14 @@
 # lexicons/ dir.
 LEXICONS = []
 
+# via https://atproto.com/specs/handle
+HANDLE_REGEX = (
```

Comment:
thanks for the link! also, fun to compare this with Bridgy Fed's which is much looser, mainly to allow unicode: https://github.com/snarfed/bridgy-fed/blob/b9be9150f5059b8bfbd0ac984ef14f109aacdc6b/common.py#L24

`DOMAIN_RE = r'^[^/:;@?!\']+\.[^/:@_?!\']+$'`